source/sigv4.c

```diff
@@ -136,8 +136,9 @@ static void intToAscii( int32_t value,
 
     /* Write base-10 remainder in its ASCII representation, and fill any
      * remaining width with '0' characters. */
-    while( lenRemaining-- > 0U )
+    while( lenRemaining > 0U )
     {
+        lenRemaining--;
```

Comment:
Why is it necessary to change the code in this way?

Reply:
This change is required because cbmc will unwind the loop k+1 times and in the last iteration, it will overflow size_t.